src/DataStream/Compression.cc

```diff
@@ -9,7 +9,11 @@
 #include <array>
 #include <cmath>
 
+#if _ARM_ // defined in cmake files
+#include "../third-party/sse2neon/sse2neon.h"
```

Comment:
Please make a symbolic link in `thirdparty/include` to `sse2neon`, so that we can simply use `#include <sse2neon/sse2neon.h>`. This is what we do with spdlog, cxxopts and other submodule dependencies